pages/about.html

```diff
@@ -0,0 +1,307 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" type="image/x-icon"
+    href="../images/favicon.ico">
+  <link rel="stylesheet"
+    href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+    crossorigin="anonymous" />
+  <link href="https://fonts.cdnfonts.com/css/cocogoose"
+    rel="stylesheet">
+  <link rel="preconnect"
+    href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="abtstyle.css">
+  <title>About | WHO Assembly </title>
+</head>
+<body>
+  
+  <div class="header-background"></div>
+
+  <div class="mobile-menu">
+    <div class="toolbar">
+      <button class="cancel" type="button"></button>
+    </div>
+    <div class="middle-part">
+      <menu class="mobile-list">
+        <li>
+          <a href="../index.html">
+            Home
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            About
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Join
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Sponsor
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            News
+          </a>
+        </li>
+      </menu>
+      <button type="button" class="campaign-button button">
+        WHO Campaign
+      </button>
+    </div>
+    <ul class="account-bar">
+      <li>
+        <a href="#">
+          <i class="fab fa-facebook-f"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fab fa-twitter"></i>
+        </a>
+      </li>
+      <li>
+        <a class="third-child" href="#">
+          English
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          My Page
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          Logout
+        </a>
+      </li>
+    </ul>
+  </div>
+
+  <!-- absolutely positioned elements -->
+
+  <header>
+
+    <div class="mobile-nav">
+      <button class="mobile-menu-button" type="button">
```

Comment:
- [ ] Your mobile menu button is not working properly on your About page. Please, check what's happening and fix this issue.